docs/user-guide.md

```diff
@@ -286,6 +287,18 @@ services:
       - db-data:/var/lib/postgresql/data
 ```
 
+- `kompose.volume.storage-class-name` defines the requests storage's class name in the PersistentVolumeClaim.
+
+For example:
+
+```yaml
+version: '3'
+volumes:
+  volume:
+    labels:
+      kompose.volume.storage-class-name: custom-storage-class-name
+```
+
 - `kompose.controller.type` defines which controller type should convert for this service
```

Comment:
docker-compose does not support lables in volumes filed. Please read the document refence: https://docs.docker.com/compose/compose-file/compose-file-v3/

Reply:
change lables to service field